web/package/agama-web-ui.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Wed Sep  4 21:00:34 UTC 2024 - Knut Anderssen <kanderssen@suse.com>
+
+- Bring back DASD management support (gh#openSUSE/1549).
+
```

Comment:
Wrong reference because a missing word,

`gh#openSUSE/agama#1549`

Reply:
A bit NP here, bit please check the PR description before merging too. Is the warning still relevant?